ci/component.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15.7
```

Comment:
The other yml files have `latest` instead of a fixed version. Should this also have?

Reply:
Cheers for spotting, the `lint.yml` wasn't using latest either so have updated that one too.